herd/tests/instructions/AArch64.mixed/M006.litmus

```diff
@@ -0,0 +1,10 @@
+AArch64 M006
```

Comment:
Do we need mixed mode for this test?

Reply:
After a quick run in Herd, no we don't - will move to non-mixed directory